src/components/molecules/PrivateRecipientSearchInput/PrivateRecipientSearchInput.tsx

```diff
@@ -21,9 +21,7 @@ const PrivateRecipientSearchInput: React.FunctionComponent<PropsType> = ({
     setSelectedUser(user);
   };
```

Comment:
I think to address issues like this, it is wiser to create a separate PR

Reply:
In this case you already changed the code in e7c05d2e2c040fd58dcda1e648598489d528fd44, so I feel like it's a valid addition to make in this PR, particularly as it's only adding a couple of lines to change, and it fixes more of the performance issues.

I won't block this PR on it though if you feel strongly otherwise.

Reply:
added to https://github.com/sparkletown/internal-sparkle-issues/issues/116